src/components/Table/hoc/withTableSettings/TableColumnSetup/TableColumnSetup.tsx

```diff
@@ -171,6 +173,8 @@ type Item = TableColumnSetupItem &
         isDragDisabled?: boolean;
     };
 
+type ItemRenderProps = Omit<Item, 'isSelected' | 'isRequired'>;
```

Comment:
This type not used anywhere and not exported. Why it needed?

Reply:
It is used in line 27 https://github.com/gravity-ui/uikit/pull/1450/files/d80cfcb5ace703089ef64e442810607f699cfda1#diff-6baaa789dea19c3913ee44fcc6ebac7f3c704b67dd7007e8e8685bd4639928c9R27